agreement/node.go

```diff
@@ -174,8 +171,39 @@ func (w *AgreementWorker) checkNodeUserInputChanges(pDevice *persistence.Exchang
 
 // get the service configuration state from the exchange, check if any of them are suspended.
 // if a service is suspended, cancel the agreements and remove the containers associated with it.
-func (w *AgreementWorker) checkServiceConfigStateChanges(exchDevice *exchange.Device) {
-	glog.V(4).Infof(logString(fmt.Sprintf("Check the service configuration state")))
+func (w *AgreementWorker) checkServiceConfigStateChanges() {
```

Comment:
Leave the call to checkServiceConfigStateChanges in the checkNodeChanges() function then you don't have to duplicate this code that gets the local device because checkNodeChanges() already does that.

Reply:
This one has to be taken out of checkNodeChanges() because the change type is `service_configstate` instead of `node`.